otter/test/convergence/test_gathering.py

```diff
@@ -212,12 +221,84 @@ def test_retry(self):
                                 next_interval=exponential_backoff_interval(2)))
 
 
-class GetScalingGroupServersTests(SynchronousTestCase):
+class GetScalingGroupServerTests(SynchronousTestCase):
```

Comment:
Non-blocking nitpick:  Should this still be "Servers" (plural), because the function name is still `get_scaling_group_servers`?